app/queries/items_by_storage_collection.rb

```diff
@@ -0,0 +1,29 @@
+module Query
+  class ItemsByStorageCollection
+    attr_reader :organization
+    attr_reader :filter_params
+
+    def initialize(organization:, filter_params:)
```

Comment:
Why pass `filter_params`? not trying to introduce any improvements, just moving as much code as simply as possible between files.

After this has been in production for a few weeks and no one's complained, we can refactor if we want.